python/pyspark/pandas/data_type_ops/boolean_ops.py

```diff
@@ -47,7 +47,7 @@ class BooleanOps(DataTypeOps):
 
     @property
     def pretty_name(self) -> str:
-        return "booleans"
+        return "bools"
```

Comment:
Rename that in order to be separate from `BooleanExtensionOps` in error messages.